shared/rakefiles/xk_infra.rake

```diff
@@ -109,7 +116,14 @@ task :apply_common_infra => [@gcp_creds_file] do
 end
 
 task :fix_common_service_account_permissions => [@gcp_creds_file] do
-  ["roles/resourcemanager.projectCreator", "roles/billing.user"].each do |role|
+  ["roles/billing.projectManager",
+   "roles/billing.user",
+   "roles/iam.organizationRoleViewer",
+   "roles/iam.serviceAccountAdmin",
+   "roles/iam.serviceAccountKeyAdmin",
+   "roles/resourcemanager.projectCreator",
+   "roles/resourcemanager.projectIamAdmin",
+   "roles/serviceusage.serviceUsageAdmin"].each do |role|
```

Comment:
Can we share one list of permissions with the above `organizations_permissions` to avoid duplication?